pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryImport.xml

```diff
@@ -912,6 +912,29 @@ public class U {
     private void g() {
         String k = C.V;
     }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>Necessary imports for @snippet tags introduced with JEP 413 in Java 18</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+/**
+ * {@snippet :
```

Comment:
Da muss ich jetzt meinen Javadoc-parser aktualisieren...

Reply:
btw. I didn't try this with the javadoc tooling. I hope, I understood the specification in the JEP correctly... :)